packages/graphql_server/src/api/openAIApi.ts

```diff
@@ -141,20 +141,20 @@ export const getCategoryFromGPT = async (topics: string[] | null, description: s
     const answer = response?.data?.choices?.[0]?.message?.content
     const categoryNumbers = answer ? convertNumbersStringToList(answer) : ['9']
     return categoryNumbers.map((categoryNumber) => {
-      return Topic[Number(categoryNumber)]
+      return Category[Number(categoryNumber)]
     })
   } catch (error) {
-    return Topic['9']
+    return [Category['9']]
   }
 }
 
 // iterating through enums needs to be done like that
 const createCategorizationPrompt = () => {
```

Comment:
this method should be rewritten completely, I think the approach is way to difficult